pyindi/www/static/js/builder-indi.js

```diff
@@ -348,7 +356,12 @@
 			// If "Enter" is pressed on writeonly area, send new text to indi
 			wo.addEventListener("keyup", (event) => {
 				if (event.key === "Enter") {
+
 					event.preventDefault() // TODO Test if needed
+					if(utilities.parentVectorIsDeleted(event.target)) {
+						//Do nothing if vector received delProperty.
+						return;
+					}
```

Comment:
This might not be needed with `pointer-events: none;` added to the css. But let's keep it in for now until more testing can be done. Same thing could be said for all of these checks on the `keyup` and `change` events.

Reply:
I justed tested deleting properties, using css to stop fieldset interaction and it works so I am removing these javascript hooks.